miner/worker.go

```diff
@@ -265,35 +265,20 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 
 		_, err := w.commitTransaction(env, tx, coinbase)
 		switch {
-		case errors.Is(err, core.ErrGasLimitReached):
```

Comment:
Moves all error cases that result in discarding transaction + future transactions from the same address via `txs.Pop()` in favor of handling via the `default` case